src/styles/_Dashboard.scss

```diff
@@ -0,0 +1,37 @@
+.card1 {
+    width: 30rem;
+    height: 15rem;
+    position: relative;
+    left: 200px;
+    top: 100px;
+}
+
+.card2 {
+    width: 30rem;
+    height: 15rem;
+    position: relative;
+    left: 200px;
+    top: 100px;
+}
+
+.card3 {
+    width: 30rem;
+    height: 15rem;
+    position: relative;
+    left: 200px;
+    top: 150px;
+}
+
+.card4 {
+    width: 30rem;
+    height: 15rem;
+    position: relative;
+    left: 200px;
```

Comment:
This styling should be combined as it is the same for each card. Was there a reason for splitting each card out?